docs/next/guides/upgrade-and-migration/migrating-from-11.1-to-12.0.md

```diff
@@ -74,14 +78,16 @@ Only the following plugins still get updated on every [`updateSettings()`](@/api
 
 #### Migrating to Handsontable 12.0
 
-If you want a plugin (e.g. a [custom plugin](@/guides/building-and-testing/plugins.md)) to still get updated every time you call [`updateSettings()`](@/api/core.md#updatesettings), use the [`afterUpdateSettings`](@/api/hooks.md#afterupdatesettings) hook to call your plugin's [`updatePlugin()`](@/api/autocolumnsize.md#updateplugin) method:
+If you want your [custom plugin](@/guides/building-and-testing/plugins.md) to still get updated on every [`updateSettings()`](@/api/core.md#updatesettings) call, set your plugin's [`SETTING_KEYS`](@/api/baseplugin.md#setting-keys) to `true`:
 
 ```js
-afterUpdateSettings() {
-  customPluginInstance.updatePlugin();
+static get SETTING_KEYS() {
+  return true;
 }
 ```
 
+To configure your custom plugin in a different way, see the [Plugins](@/guides/building-and-testing/plugins.md) guide.
```

Comment:
I would rewrite this as:

However, it is most often a better idea to provide an explicit list of configuration options observed by the plugin. See the [Plugins](@/guides/building-and-testing/plugins.md) guide for details.